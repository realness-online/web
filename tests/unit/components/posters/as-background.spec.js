import { shallowMount } from '@vue/test-utils'
import as_background from '@/components/posters/as-background'
describe('@/components/posters/as-fill-figure.vue', () => {
  describe('Renders', () => {
    it('a new background for a poster without one', () => {
      const wrapper = shallowMount(as_background)
      expect(wrapper.element).toMatchSnapshot()
    })
  })
})
