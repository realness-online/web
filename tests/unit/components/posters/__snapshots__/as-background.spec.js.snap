// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`@/components/posters/as-fill-figure.vue Renders a new background for a poster without one 1`] = `
<rect
  fill="white"
  height="100%"
  itemprop="background"
  width="100%"
/>
`;
